source/user-manual/capabilities/active-response/ar-use-cases/wazuh-with-yara.rst

```diff
@@ -0,0 +1,423 @@
+
+It defines the criteria used to execute a specific command:
+
+- The ``command`` to execute. It references the ``yara`` command created above.
+
+- The ``location`` setting as ``local``. It means that the active response is executed on the agent that generates the alert.
+
+- You can write a list of rule IDs that will trigger the active response in the ``rules_id`` setting. This example uses rule ``550``, new file added to the system, and rule ``554``, file modified in the system.
+
+Rules and decoders
+^^^^^^^^^^^^^^^^^^
+
+Define a set of rules and decoders to trigger alerts from the events generated by the YARA active response.
+
+Create a decoder file, for example ``/var/ossec/etc/decoders/yara_decoders.xml``, and add the following:
+
+.. code-block:: none
+
+  <!--
+   - YARA decoders
+   - Created by Wazuh, Inc.
+   - Copyright (C) 2015-2021, Wazuh Inc.
+   - This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+  -->
+ 
+  <decoder name="yara">
+    <prematch>wazuh-yara: </prematch>
+  </decoder>
+
+  <decoder name="yara">
+    <parent>yara</parent>
+    <regex>wazuh-yara: (\S+) - Scan result: (\S+) (\S+)</regex>
+    <order>log_type, yara_rule, yara_scanned_file</order>
+  </decoder>
+
+  <decoder name="yara">
+    <parent>yara</parent>
+    <regex>wazuh-yara: (\S+) - Yara active response error. (\.+)</regex>
+    <order>log_type, error_message</order>
+  </decoder>
+
+
+Similarly create a rule file, ``/var/ossec/etc/rules/yara_rules.xml``, with the following content:
+
+.. code-block:: none
+
+  <!-- 
+   - YARA rules 
+   - Created by Wazuh, Inc. 
+   - Copyright (C) 2015-2021, Wazuh Inc. 
+   - This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2. 
+  -->
+
+   <group name="yara,">
+      <rule id="100100" level="0">
+          <decoded_as>yara</decoded_as>
+          <description>YARA rules grouped.</description>
+      </rule>
+
+      <rule id="100101" level="5">
+          <if_sid>100100</if_sid>
+          <field name="error_message">\.+</field>
+          <description>YARA error detected.</description>
+      </rule>
+
+      <rule id="100102" level="10">
+          <if_sid>100100</if_sid>
+          <field name="yara_rule">\.+</field>
+          <description>YARA $(yara_rule) detected.</description>
+      </rule>
+  </group>
+
+Restart the Wazuh manager for the changes to take effect. 
+
+Wazuh agent configuration
+-------------------------
+
+The following section assumes YARA is already installed on the monitored endpoint.  If you need to install it, you can do it following the `official YARA installation guide <https://yara.readthedocs.io/en/stable/gettingstarted.html#compiling-and-installing-yara>`_. 
+
+The script configured to run as part of the active response settings defined on the Wazuh manager, ``yara.sh``, needs to be placed under ``/var/ossec/active-response/bin`` on the Wazuh agent side. Add the following content to it:
```

Comment:
```suggestion
The script is configured to run as part of the active response settings defined on the Wazuh manager, ``yara.sh``, needs to be placed under ``/var/ossec/active-response/bin`` on the Wazuh agent side. Add the following content to it:
```

Reply:
The sentence is correct as it is, it refers to the script configured to run in the active response and states that it must be placed under ``/var/ossec/active-response/bin`` on the Wazuh agent side.